notifications.bs

```diff
@@ -189,6 +212,24 @@ or vibration pattern that is not otherwise accessible to the end user.
   <li><p>If <var>options</var>'s <code>sticky</code> is true, set
   <var>notification</var>'s <a>sticky preference flag</a>.
 
+  <li><p>Set <var>notification</var>'s list of <a>actions</a> to an empty list,
```

Comment:
Hmm, I guess it's fine to leave this for now then. Thank you for the research. Should probably be cleaned up at some point, but given the precedent it's not really important to do it as part of this.